src/components/chat/ChatList.tsx

```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components';
+
+import { ChatRoomSummary } from '../../types';
+
+import ChatListRow from './ChatListRow';
+
+interface ChatListProps {
+  chatRooms : ChatRoomSummary[]
+}
+
+export default function ChatList({ chatRooms } : ChatListProps) {
+  if (!chatRooms.length) {
```

Comment:
p3: 어쩌면 사소할수도 있지만 저는 중요하다고 생각하는 부분인데요. 조건식을 작성할 때 boolean 타입의 값을 제외한 truthy, falsy값들은 명시적으로 작성해주는게 좋다고 생각해요. 생각보다 이와 관련된 문제로 에러가 발생했을 때 해결하기까지 어려움이 많아요.

```suggestion
  if (chatRooms.length === 0) {
```